src/databricks/labs/ucx/workspace_access/manager.py

```diff
@@ -148,6 +149,13 @@ def _save(self, items: list[Permissions]):
 
     def load_all(self) -> list[Permissions]:
         logger.info(f"Loading inventory table {self._full_name}")
+        if self._table not in [d.tableName for d in self._fetch(f"SHOW TABLES IN {self._catalog}.{self._schema}")]:
```

Comment:
And what user would be doing with this message? :)

Reply:
@nfx I thought the installer only creates the job. Its the setup schema in the assessment job which creates the schema first and each task create the tables

Reply:
@HariGS-DB He meant to say "soon it will" :) See https://github.com/databrickslabs/ucx/pull/474/files#diff-67e29ef6d5123a708ed3144d670ba229c7d64995aae2d58f2ac9f7b69e6c88d5R106 in PR #474

Reply:
i see. understood. will add the changes and push it

Reply:
I have done the changes to look for table rows and updated the error msg to be more descriptive. I noticed the actual RunTimeBackend fetch returns a list(even though the func signature says iter) whereas the MockBackend returns an iter